Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+
+Core Rules
+
+[color=#a4885c]0.[/color] Take these rules as they are to be intended and not as they are written literally. Rule skirting and lawyering is also bannable at the admin teams discretion.
+[color=#a4885c]1.[/color] Don’t be a dick. We’re all here to have fun. Antagonists have leeway here since ruining the round is kind of their job, but try to keep it interesting if you’re antag.
+    - Be considerate of what other players want. If everyone is dead, don’t stall the end of the round to finish your personal projects (such as building a shuttle).
+[color=#a4885c]2.[/color] These are English servers. Do not speak other languages in IC or OOC, our staff cannot moderate it and you will be asked to stop.
+[color=#a4885c]3.[/color] These servers are “Roleplay Expected”, meaning you should remain in character at all times unless speaking in OOC chat or local OOC (LOOC) chat. Basic roleplaying standards like not using emoji/emoticons or textspeak (:), :D, xD, lol, etc.) in speech, and using real names for characters, are expected to be followed.
+[color=#a4885c]4.[/color] Follow basic common sense–you cannot murder someone because they slipped you on a banana.
+    - This isn’t a deathmatch, be humane.
+    - What is considered a reasonable reason to escalate conflict is listed below under ESCALATION RULES and is at the discretion of admins.
+    - Declaring a revolution or going on strike is an antagonist only activity. “Cargonia” and other departmental revolutions are prohibited unless admin permission is obtained.
+[color=#a4885c]5.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color]. This includes any direct or implied sexual acts as well as explicit mentions of sexual behavior.
+[color=#a4885c]6.[/color] Harassment of other players for OOC/meta reasons is [color=#ff0000]not allowed[/color].
+	- Annoying somebody in a round for IC/RP reasons is fine. Doing it consistently across multiple rounds for OOC reasons (“I don’t like this player” AKA metagrudging) to the point where it becomes seriously problematic for the other player(s) is [color=#ff0000]not[/color].
+
+[color=#a4885c]7.[/color] Hate speech, slurs and bigotry are not allowed.
+	- Intentionally seeking to demean others due to their actual or perceived race, sex, gender, sexual orientation or the like is not tolerated.
+	- Words that are closely tied to real life slurs are [color=#ff0000]not allowed[/color].
+    - Species-ism (IC or OOC bigotry against other in-game species) is also NOT ALLOWED. Not even for roleplay or because it's your "character".
+
+[color=#a4885c]8.[/color] Offensive/bad/out-of-character character names and descriptions put you at the mercy of administrators, from receiving brain damage to becoming valid to straight up getting banned on the spot depending on how bad they are.
+    - Character names should be something that a member of that species would reasonably have.
```

Comment:
Do we even have a species naming policy / does the random name generator support species?